_harp/academy/_prerequisites.md

```diff
@@ -1,63 +1,62 @@
 # Course prerequisites
 
-Before you join us, you will be expected to complete all the material below. If you have any doubts about your ability to complete any of it, please contact us before the start of the course.
+We have chosen these resources as our entrance criteria. If you are not comfortable coding at this standard by the time the course starts you may struggle. These tasks are part of how we will measure whether you are a good candidate for the course as well as whether you can learn in the collaborative environment here. You do not need to have completed these prerequisites before you send in your application, but you should at least have made a good start on them. 
 
-## Code
+**Complete Beginners**  
+If you are completely new to coding, run through these Codecademy tracks first (in this order).
+* [HTML & CSS](https://www.codecademy.com/learn/web)
+* [Make a Website](https://www.codecademy.com/en/skills/make-a-website)
+* [Javascript](https://www.codecademy.com/learn/javascript)
+* [Make an Interactive Website](https://www.codecademy.com/en/skills/make-an-interactive-website) (?? find a better one)
```

Comment:
Is this to go live on the site? We should remove 'Make an interactive site' because it contains jQuery